runtime/cmake/platform/os/zos.cmake

```diff
@@ -25,7 +25,10 @@ set(CMAKE_SHARED_LIBRARY_PREFIX "lib")
 
 set(CMAKE_IMPORT_LIBRARY_PREFIX "lib")
 
-list(APPEND OMR_PLATFORM_COMPILE_OPTIONS
-    "\"-Wc,inline(auto,noreport,600,5000)\""
-    "\"-Wc,list(),offset,gonumber\""
-)
+# todo: perhaps this belongs in corresponding toolchain file?
+if($(TOOLCHAIN_TYPE),xlc)
+    list(APPEND OMR_PLATFORM_COMPILE_OPTIONS
+        "\"-Wc,inline(auto,noreport,600,5000)\""
+        "\"-Wc,list(),offset,gonumber\""
```

Comment:
https://github.ibm.com/runtimes/openj9-openjdk-jdk21-zos/issues/320
Opened an issue as this might be a more extended discussion and is not directly supported. We might need to communicate with the Open XL team the necessity of these, or if there are equivalent alternatives.